### ISSUE TITLE
Automatically increase Java/JVM resource usage when dealing with large model dumps

### ISSUE THREAD

**Issue by vruusmann** [jpmml/sklearn2pmml#330]
The Python wrapper should consider the following before launching the Java executable:

1. Size of the model dump file (1 MB, 10 MB, 100 MB).
2. Possibly, the complexity of the model object inside the dump file - is it feature-heavy (eg. linear regression with very many features), model-heavy (eg. ensemble model with few features), or both (eg. ensemble model with very many features)?
3. Available system resources. How much RAM is physically available? How much of this RAM is currently free for the Java executable to claim?

TLDR: Automatically adding appropriate `-Xms` and `-Xmx` values, in order to make sure that the PMML converter performance won't be visibily memory-bound.
